pkg/apis/core/v1alpha1/filewatch_types.go

```diff
@@ -86,9 +94,31 @@ func (in *FileWatch) IsStorageVersion() bool {
 	return true
 }
 
-func (in *FileWatch) Validate(ctx context.Context) field.ErrorList {
-	// TODO(user): Modify it, adding your API validation here.
-	return nil
+func (in *FileWatch) Validate(_ context.Context) field.ErrorList {
```

Comment:
ooh, love this

Reply:
Was super happy that it just worked out of the box!